src/logic/safe/store/actions/transactions/fetchTransactions/loadGatewayTransactions.ts

```diff
@@ -100,23 +90,13 @@ export const loadPagedQueuedTransactions = async (
 
 export const loadQueuedTransactions = async (safeAddress: string): Promise<QueuedGatewayResponse['results']> => {
   const queuedTransactionsUrl = getQueuedTransactionsUrl(safeAddress)
-  try {
-    const {
-      data: { results, ...pointers },
-    } = await axios.get<QueuedGatewayResponse, AxiosResponse<QueuedGatewayResponse>>(queuedTransactionsUrl)
-
-    if (!queuedPointers[safeAddress] || queuedPointers[safeAddress].next === null) {
-      queuedPointers[safeAddress] = pointers
-    }
-
-    return results
-  } catch (error) {
-    // When the safe is just created there is a delay until the gateway recognize the
-    // safe address, when that happens it returns 404.
-    if (error.response.status === 404) {
-      return []
-    }
-
-    throw Error(`There was an error trying to fetch queued txs from safeAddress ${safeAddress}`)
+  const {
+    data: { results, ...pointers },
+  } = await axios.get<QueuedGatewayResponse, AxiosResponse<QueuedGatewayResponse>>(queuedTransactionsUrl)
```

Comment:
```suggestion
  } = await axios.get<QueuedGatewayResponse>(queuedTransactionsUrl)
```

type sig for axios.get is `get<T = any, R = AxiosResponse<T>>(url: string, config?: AxiosRequestConfig): Promise<R>;`